src/test/java/org/mule/extensions/jms/test/util/BadProxy.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extensions.jms.test.util;
+
+import eu.rekawek.toxiproxy.Proxy;
+import eu.rekawek.toxiproxy.ToxiproxyClient;
+
+import java.io.IOException;
+
+public class BadProxy {
```

Comment:
maybe this could be exported into a standalone mule-plugin so it can be reused elsewhere, since many other components may need to test similar scenarios.